### PR TITLE
Task1/tests

### PR DESCRIPTION
**Pull request by maria-guzik**
Fixed the issues that prevented the tests from running: 
changed the columns chosen in poincare_tests.py
made several changes in spectral.py:
- line 26: fixed module name from scisignal to sc as scipy.signal was imported as sc
- line 46: fixed the index used for creating the periodogram as first_index[-1] was larger than second_index[0], resulting in an empty periodogram, to get the periodogram with values between cuts it is now first_index[0]:second_index[0] (first instance of a value being larger than cut 1 to the first instance of a value being larger than cut 2)
- lines 53 and 54: changed the format of multiplication to prevent TypeError resulting from multipling the sequence by a float.

Moved test files for easier testing and created an additional test file to better understand the code  

Changed the modules from SciPy to Numpy to avoid depreciation, as many of the used functions will be removed in SciPy 2.0.0.

### CODE REVIEW

File: poincare_tests.py [hunk 1]
```diff
@@ -6,7 +6,8 @@
 class TestPoincare(unittest.TestCase):
 
     def setUp(self):
-        self.signal_real1 = Signal("RR1.csv", 1, 2, annotation_filter=(2,))
+        #changed the column indexes from 1 and 2 to 0 and 1 to match python indexing
+        self.signal_real1 = Signal("RR1.csv", 0, 1, annotation_filter=(2,))
         self.signal_real1.set_poincare()
```
Comment (jaropis) [inline poincare_tests.py:11]: Please check if this is correct. This version of the software worked with *.rea files which had a time track as the first column. Perhaps RR1.csv is not the best file to test it.

File: signal_properties/spectral.py [hunk 2]
```diff
@@ -43,15 +43,15 @@ def get_bands(self, cuts, df):
...
                 first = second
             elif len(first_index) >= 1: # so, there is no power in the band above - is there any power
                 power_in_bands.append(sum(self.periodogram[first_index[0]:first_index[-1]]))
                 break
             else:
                 break
-        print("dupa", power_in_bands * df)
-        return scipy.array(power_in_bands) * df
+        print("dupa", [i * df for i in power_in_bands])
+        return scipy.array([i * df for i in power_in_bands])
```
Comment (jaropis) [inline signal_properties/spectral.py]: The print statements were there for debugging purposes. Please remove them, especially if they are so offensive ;)

**Review by jaropis — COMMENTED** [review]
Please make sure that the package works with rea files (prof. Guzik should have some) rather than the RR1.csv file. Also please close this review, branch off the "development" branch from the master branch, and make another PR to the development branch, rather than master.

Overall this is great work! I am happy somebody is finally working on this code and doing great work at it!

**Comment by maria-guzik** [comment]
Thank you, I applied the changes, I left the poincare_tests.py with the RR.csv file since it was originally used there and all the tests compare the values with the ones specific for RR.csv.

**Pull request closed without merging**